dataactbroker/fileRoutes.py

```diff
@@ -192,7 +192,7 @@ def delete_submission(submission):
         """ Deletes all data associated with the specified submission
         NOTE: THERE IS NO WAY TO UNDO THIS """
 
-        if submission.publish_status_id == PUBLISH_STATUS_DICT['published']:
+        if submission.publish_status_id != PUBLISH_STATUS_DICT['unpublished']:
             return JsonResponse.error(ValueError("Certified submissions cannot be deleted"), StatusCode.CLIENT_ERROR)
```

Comment:
We may want to update this message. Can be misleading if someone sees "Updated" as the publish status and the submission status no longer shows up as "Certified" on the front-end.